com/laboon/WorldTest.java

```diff
@@ -0,0 +1,117 @@
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class WorldTest {
+
+
+	
+	//Creates a 10x10 grid and tests to see if the two toString methods return the same result.
+	@Test
+	public void testMap() {
+		World bob = new World(10, 10, 50);
+		
+	
+		StringBuilder first = new StringBuilder();
+		first.append("  ");
+		first.append(bob.toString());
+		String second = bob.OriginaltoString();
```

Comment:
This will fail as soon as we get rid of the old toString, and why would we keep it around?

Remember that the purpose of automated tests is to have a test suite which will continue to work as the system evolves.

Should be explicitly checking against expected behavior.